kernel/kernel-impl/src/main/java/org/sakaiproject/email/impl/BasicEmailService.java

```diff
@@ -1325,6 +1325,10 @@ protected void setContent(String content, List<Attachment> attachments, MimeMess
 				// attach the file to the message
 				MimeBodyPart mbp = createAttachmentPart(attachment);
 				int mbpSize = mbp.getSize();
+				if (mbpSize < 0)
+				{
+					log.warn("Unexpected MIME body part size: " + mbpSize);
```

Comment:
You should use log.warn("Unexpected MIME body part size: {}", mbpSize)

That way if this was ever made a debug and debug wasn't enabled at any time, there'd be less evaluation cost. Or if we only logged at levels above WARN.